app/services/idv/steps/doc_auth_base_step.rb

```diff
@@ -22,18 +22,32 @@ def attempter_throttled?
         Throttle.for(**idv_throttle_params).throttled?
       end
 
+      def attempts_remaining
+        Throttle.for(**idv_throttle_params).remaining_count
```

Comment:
Between this and `attempter_throttled?`, I see we're querying the database twice now when throttled. Maybe we could create a memoized `def throttle` method like we do elsewhere?

```rb
      def throttle
        @throttle ||= Throttle.for(
          user: current_user,
          throttle_type: : idv_resolution,
        )
      end
```

At that point, I might just recommend getting rid of all these methods `idv_throttle_params`, `attempter_increment`, `attempter_throttled?`, `attempts_remaining` and replace with `throttle.increment`, `throttle.throttled?` and `throttle.remaining_count`.

Reply:
Done in 4f67529